package.json

```diff
@@ -24,7 +25,9 @@
     "abcolor": "^0.5.5",
     "c3": "^0.4.15",
     "classnames": "^2.2.5",
+    "dota2-emoticons": "^1.0.1",
```

Comment:
move to devDependencies?

Reply:
It'll cause eslint error `'dota2-emoticons' should be listed in the project's dependencies, not devDependencies. (import/no-extraneous-dependencies)` since `conmonents/Match/Chat/Chat.jsx` imports json from dota2-emoticons 
https://github.com/odota/ui/pull/1154/files#diff-b19c36694d7399b5d01a371f4b2dff11R12

Reply:
ah, ok

fs-extra should probably be moved though (disable the lint warning in the file)